app/assets/stylesheets/application/styles/_event_show.scss

```diff
@@ -198,10 +198,10 @@
   text-align: center;
 
   .stripe-button-el {
+    @extend .event-show__booking-button;
```

Comment:
Name of placeholder `.event-show__booking-button` should be written in all lowercase letters with hyphens instead of underscores